logmerge.py

```diff
@@ -263,24 +267,16 @@ def expand_paths(paths, suffix):
     path_sizes = {}
 
     for path in globbed:
-        if path.endswith(".zip"):
+        if zipfile.is_zipfile(path):
             zf = zipfile.ZipFile(path, 'r')
             for info in zf.infolist():
-                zpath = path + "/" + info.filename
+                zpath = path + ":" + info.filename
                 rv.append(zpath)
                 total_size += info.file_size
-                path_sizes[zpath] = info.file_size
+                path_sizes[info.filename] = info.file_size
         else:
             rv.append(path)
-
-            zip_suffix = path.find(".zip/")
```

Comment:
ooh, i think this was here for an important case... wondering if you saw some reason why this needed changing?  If it's because "it looked weird", that may be true, but it functioned correctly, iirc